compliance_tool/cli.py

```diff
@@ -1,57 +1,67 @@
 import argparse
-from compliance_tool import list, scan, remediate
+from compliance_tool import list_, scan, remediate
 
 
-def prepare_parser_list(subparsers) -> None:
-    parser = subparsers.add_parser("list",
-                                   help="list available components")
-    parser.set_defaults(func=list.execute)
+def prepare_parser_list(cli_sub_parsers, global_parser, profile_parser) -> None:
+    parser = cli_sub_parsers.add_parser("list", parents=[global_parser],
+                                        help="list available components")
+    parser.set_defaults(func=list_.execute)
     list_subparsers = parser.add_subparsers(required=True)
 
     list_subparsers.add_parser("profiles",
                                help="list available profiles")
 
-    parser = list_subparsers.add_parser("controls",
-                                        help="list available controls for a given profile")
-    parser.add_argument("--profile")
+    list_subparsers.add_parser("controls", parents=[profile_parser],
+                               help="list available controls for a given profile")
```

Comment:
Do you prefer "controls" over "requirements" and why?

Reply:
I'm on the fence here. We use the term `control` a lot and it is shorter. But formally it is a `requirement`.

@Mab879 @marcusburghardt Opinions? Should it be `--requirement` dubbed with control in the help message?

Reply:
Formally they are are `requirement`, but I think control might be a better fit here.

Reply:
I also feel that "control" is better here, control is a term that people expect in this context.